rasa/cli/arguments/default_arguments.py

```diff
@@ -27,6 +27,18 @@ def add_model_param(
         )
 
 
+def add_no_plot_param(
```

Comment:
how about moving this to `rasa.cli.arguments.test`?

Reply:
I totally understand the rationality here, but we already have test.py in this folder, and currently I'm a little bit overwhelmed by amount of word "test" used around the codebase, I don't want to introduce new sub-folder called the same.
I don't have a good enough suggestion for refactoring this though.

Reply:
I think Tobias meant placing it in test.py (makes sense since it's the only place where it is used)

Reply:
oh right! Will do